include/benchmark/benchmark_api.h

```diff
@@ -153,6 +153,8 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 #include <stddef.h>
 #include <stdint.h>
 
+#include <vector>
```

Comment:
No issue. So long as STL stuff doesn't affect the timing results then I think we should encourage it's use.